src/contribute/work-in-progress.md

```diff
@@ -25,7 +25,7 @@ Current branches include:
 | Feature | Branch | JIRA Component | More Info |
 | ---- | ---- | ---- | ---- |
 | Apache Gearpump Runner | [gearpump-runner](https://github.com/apache/beam/tree/gearpump-runner) | [runner-gearpump](https://issues.apache.org/jira/browse/BEAM/component/12330829) | [README](https://github.com/apache/beam/blob/gearpump-runner/runners/gearpump/README.md) |
-| Python SDK | [python-sdk](https://github.com/apache/beam/tree/python-sdk) | [sdk-py](https://issues.apache.org/jira/browse/BEAM/component/12328910) | [README](https://github.com/apache/beam/blob/python-sdk/sdks/python/README.md) |
+| Python SDK | [python-sdk](https://github.com/apache/beam/tree/master/sdks/python) | [sdk-py](https://issues.apache.org/jira/browse/BEAM/component/12328910) | [README](https://github.com/apache/beam/blob/master/sdks/python/README.md) |
```

Comment:
I think there is value in keeping that README. It contains folder specific information. Or we can merge that information back to top level README. I prefer the first option, but I think both makes sense.